src/components/PosterByline.tsx

```diff
@@ -1,28 +1,52 @@
 import * as React from "react";
+import { graphql } from "relay-runtime";
 import Image from "./Image";
+import { useFragment, useQueryLoader } from "react-relay";
+import type { PosterBylineFragment$key } from "./__generated__/PosterBylineFragment.graphql";
+import Hovercard from "./Hovercard";
+import PosterDetailsHovercardContents from "./PosterDetailsHovercardContents";
+import { PosterDetailsHovercardContentsQuery } from "./PosterDetailsHovercardContents";
+import type { PosterDetailsHovercardContentsQuery as HovercardQueryType } from "./__generated__/PosterDetailsHovercardContentsQuery.graphql";
+const { useRef } = React;
+
+export const PosterBylineFragment = graphql`
```

Comment:
Should be in a separate file, If you can follow spreadd structure or 
src
 queries
 mutation